src/raw_client.rs

```diff
@@ -1190,6 +1191,25 @@ mod test {
         assert_eq!(resp.hash_function, Some("sha256".into()));
         assert_eq!(resp.pruning, None);
     }
+
+    #[test]
```

Comment:
Thanks for adding this. I haven't vetted the reliability of this server and so would prefer to ignore it for unit tests.

```suggestion
    #[test]
    #[ignore = "depends on a live server"]
```

Reply:
Added the ignore statement into the latest commit.